infrastructure/models.py

```diff
@@ -224,6 +315,122 @@ def get_absolute_url(self):
+        help_text='Select or create named fuel categories.'
+    )
+    def __str__(self):
+        return self.name
+
+class FuelCategory(Publishable):
+    name = models.CharField(max_length=140)
+
+    class Meta:
+        verbose_name_plural = 'fuel categories'
+
+    def __str__(self):
+        return self.name
+
+
+class PowerPlant(Publishable):
+    """Describes a Power Plant"""
+    name = models.CharField(max_length=140)
+    slug = models.SlugField(max_length=150, allow_unicode=True)
+    infrastructure_type = models.ForeignKey(
+        InfrastructureType,
+        models.SET_NULL, blank=True, null=True,
+        help_text='Select or create named insfrastructure types.'
+    )
+    fuel = models.ManyToManyField(
+        'Fuel', 
+        blank=True, 
+    )
+    countries = models.ManyToManyField('locations.Country', blank=True)
+    regions = models.ManyToManyField(
+        'locations.Region',
+        blank=True,
+        help_text='Select or create geographic region names.'
+    )
+    status = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=ProjectStatus.STATUSES
+    )
+    plant_year_online = models.PositiveSmallIntegerField(blank=True, null=True)
+    plant_month_online = models.PositiveSmallIntegerField(blank=True, null=True)
+    plant_day_online = models.PositiveSmallIntegerField(blank=True, null=True)
+    
+    @property
+    def fuzzy_plant_online_date(self):
+        return fuzzydate(self.plant_year_online, self.plant_month_online, self.plant_day_online)
+    
+    decommissioning_year = models.PositiveSmallIntegerField(blank=True, null=True)
+    decommissioning_month = models.PositiveSmallIntegerField(blank=True, null=True)
+    decommissioning_day = models.PositiveSmallIntegerField(blank=True, null=True)
+
+    @property 
+    def fuzzy_decommissioning_date(self):
+        return fuzzydate(self.decommissioning_year, self.decommissioning_month, self.decommissioning_day)
+
+    plant_capacity = models.BigIntegerField(
+        blank=True, null=True,
+    )
+    plant_output = models.BigIntegerField(
+        blank=True, null=True,
+    )
+
+    plant_output_unit = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=ProjectPlantUnits.UNITS
+    )
+
+    plant_output_year = models.PositiveSmallIntegerField(blank=True, null=True)
+    estimated_plant_output = models.BigIntegerField(
+        blank=True, null=True
+    )
+    estimated_plant_output_unit = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=ProjectPlantUnits.UNITS
+    )
+
+    plant_CO2_emissions = models.BigIntegerField(
+        blank=True, null=True
+    )
+
+    plant_CO2_emissions_unit = models.PositiveSmallIntegerField(
+        blank=True, null=True,
+        choices=ProjectPlantUnits.UNITS
+    )
+
+    grid_connected = models.NullBooleanField('Grid connected?')
+    latitude = models.DecimalField(max_digits=9, decimal_places=6)
```

Comment:
Some power plants in the CSV sheet have empty latitude and longitude values, so I think we will have to make these `blank=True, null=True` also